docs/pages/clients/compatibility.md

```diff
@@ -2,20 +2,12 @@
 title: Compatibility
 ---
 
-Expo Development Client contains three packages, which have to be compatible with each other. To check which versions are compatible, check the tables below.
-
-## expo-dev-menu
-
-| expo-dev-menu-interface | expo-dev-menu     |
-| ----------------------- | ----------------- |
-| `0.2.0`                 | `0.4.X`           |
-| `0.1.2`                 | `0.3.X`           |
-| `0.1.0` - `0.1.1`       | `0.1.X` - `0.2.X` |
-
-## expo-dev-launcher
-
-| expo-dev-menu-interface | expo-dev-launcher |
-| ----------------------- | ----------------- |
-| `0.2.0`                 | `0.3.X`           |
-| `0.1.2`                 | `0.2.X`           |
-| `0.1.0` - `0.1.1`       | `0.1.X`           |
+Expo Development Client contains three packages, which have to be compatible with each other. To check which versions are compatible, check the table below.
+
+| expo-dev-menu-interface | expo-dev-menu     | expo-dev-launcher |
```

Comment:
This is definitely easier to parse, but since dev's will now be installing with expo-dev-client and a compatible version of all of the packages will be installed, is it still needed.

We do have a compatibility issue with expo-updates(-interface) so a compatibility table for that instead would be useful https://linear.app/expo/issue/ENG-1381/update-compatibility-page-in-dev-client-docs